### PR TITLE
Enhancement: Add `Makefile`

### PR DESCRIPTION
**Pull request by localheinz**
This pull request

- [x] adds a `Makefile`

💁‍♂️ Run

```shell
make
```

or

```shell
make help
```

to show a list of available commands.

Run

```shell
make tests
```

to run the existing tests.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.PHONY: it
+it: tests ## Runs the tests target
```
Comment (cmb69) [inline Makefile:2]: Not sure if we should have that. I find `make` as shortcut for `make help` quite nice.

**Review by cmb69 — COMMENTED** [review]
Thank you for the PR! This makes sense to me. (Although Windows users won't be able to use it, but I assume that isn't a real problem.)

File: Makefile (hunk 1 above)
Comment (localheinz) [reply Makefile]: @cmb69 

Adjusted!

**Review by cmb69 — APPROVED** [review]
Thank you!

**Comment by localheinz** [comment]
@cmb69 

Once this has been merged, I can update #559 - or the other way around!

**Comment by cmb69** [comment]
Thanks for this PR! I'll have a look at #559 soon.

**Comment by localheinz** [comment]
Thank you, @cmb69

**Pull request merged**